bcipy/simulator/task/task_runner.py

```diff
@@ -126,12 +131,15 @@ def main():
         task_factory = TaskFactory(params_path=sim_args['parameters'],
                                    source_dirs=sim_args['data_folder'],
                                    signal_model_paths=sim_args['model_path'],
-                                   sampling_strategy=classify(sim_args['sampler']),
-                                   task=SimulatorCopyPhraseTask)
+                                   sampling_strategy=classify(
+                                       sim_args['sampler']),
+                                   task=SimulatorCopyPhraseTask,
+                                   sampler_args=json.loads(sim_args['sampler_args']))
```

Comment:
I'd make this a method. We could add some parameter validation or helpful logging to it.